src/Storefront/Theme/Exception/ThemeException.php

```diff
@@ -59,12 +59,12 @@ public static function couldNotFindThemeById(string $themeId): self
         );
     }
 
-    public static function InvalidScssValue(mixed $value, string $type, string $name): self
+    public static function InvalidScssValue(mixed $value, string $type, string $name = ''): self
```

Comment:
should we mark the name as deprecated and remove it with the next major?

Reply:
Good catch. I actually forgot to revert this. The name is still necessary in the parameters.